### PR TITLE
Update readme.md

### PR DESCRIPTION
**Pull request by jpellissari**
Improve to installation section of readme.md

### CODE REVIEW

**Comment by barryvdh** [comment]
Why is the second method preferred? Especially when you copy the binary to a different path.

**Comment by jpellissari** [comment]
It's just a personal opinion. I can remove the prefer part.

**Comment by jpellissari** [comment]
There you go. I removed the personal opinion part! Let me know if there is any other improvement.

File: readme.md [hunk 1]
```diff
@@ -4,36 +4,67 @@
...
+### Wkhtmltopdf Installation
 
-The package provides $app['snappy.pdf'] and $app['snappy.image']. You have to set the binary location in the config file. Copy `config/snappy.php` to your own config, or use your ConfigServiceProvider so set the config keys.
+Choose one of the following options to install wkhtmltopdf/wkhtmltoimage.
 
-and then adapt the "binary" line in the published config file (afer publishing should be present in: app/config/packages/barryvdh/laravel-snappy/config.php).
+1. Download wkhtmltopdf from [here](http://wkhtmltopdf.org/downloads.html); 
+2. Or install as a composer dependencie. See [wkhtmltopdf binary as composer dependencies](https://github.com/KnpLabs/snappy#wkhtmltopdf-binary-as-composer-dependencies) for more information.
```
Comment (barryvdh) [inline readme.md]: *dependency

**Comment by jpellissari** [comment]
Fixed

**Pull request merged**